testplan/docs/troubleshooting/rancher.md

```diff
@@ -0,0 +1,43 @@
+---
+id: rancher
+title: Rancher Troubleshooting
+sidebar_label: Rancher
+---
+
+## Issues
+
+### Issue1: Openebs installation failing on cluster2
+
+#### What is the issue?
+
+In the Rancher pipeline, the installation of OpenEBS via Kubera OnPrem Director was stuck. On investigating more we found out that openebs-upgrade and cstorpoolauto pod present in maya-system namespace was showing the following error -
```

Comment:
Timestamp mismatch between master and nodes component was the actual root cause for metrics-server kube-apiservice to not perform. Those details are to be included for better understanding in future

Reply:
Good point @gprasath. I'll add that.